src/components/DashboardBody.js

```diff
@@ -49,20 +49,26 @@ const dashboardBodyProps = {
     },
   ]
 }
+const BuildWidgets = widgetArray => {
+
+  return widgetArray.map( widget => {
+    return  {
+      'Text': <Text { ...widget } key={widget.title} />,
+      'Image': <Image { ...widget } key={widget.title} />,
+      'QRCode': <QRCode { ...widget } key={widget.title} />,
+      'Clock': <Clock { ...widget } key={widget.title} />,
+    }[widget.type]
```

Comment:
Please add spaces between opening and closing parentheses and braces, and arguments, in a function invocation, control flow expression, or array or object dereference:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
{a:b}['a']
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
{ a: b }[ 'a' ]
```